tests/js/client/shell/api-gharial-creation-cluster-spec.js

```diff
@@ -52,9 +47,9 @@ describe('General graph creation', function () {
 
   const clean = () => {
     try {
-      request["delete"](`/_api/gharial/${gn}?dropCollections=true`);
+      arango.DELETE_RAW(`/_api/gharial/${gn}?dropCollections=true&waitForSync=true`);
     } catch (ignore) {}
-
+    db._flushCache();
```

Comment:
this is arangosh internal, since the tests modify the server by raw http requests, arangosh must flush its caches in order to successfully tear down using its own .js-API.